### PR TITLE
Create CODE_OF_CONDUCT.md

### LINKED ISSUE #10: Create a code of conduct
The contributing guide references to a code of conduct that we don't have yet.

Should we make an organization wide one?

### PR DESCRIPTION
**Pull request by MathewBiddle**
closes #10

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+individual, or aggression toward or disparagement of classes of individuals.
+
+**Consequence**: A permanent ban from any sort of public interaction within
+the community.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
```
Comment (ocefpaf) [inline CODE_OF_CONDUCT.md]: I'm not sure what are the differences but there is a 2.1 version: https://www.contributor-covenant.org/version/2/1/code_of_conduct/

Comment (MathewBiddle) [reply CODE_OF_CONDUCT.md]: Shoot. I was going to copy the 2.1 version but then GH offered to to create this one automagically. I can't find a diff between the two :(

Comment (ocefpaf) [reply CODE_OF_CONDUCT.md]: Looks like this is the only diff:

```diff
-We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation.
```

**Comment by MathewBiddle** [comment]
should be updated to v2.1 now

**Comment by MathewBiddle** [comment]
okay, now it should be good to review.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -52,15 +52,15 @@ decisions when appropriate.
...
+Examples of representing our community include using an official email address,
 posting via an official social media account, or acting as an appointed
 representative at an online or offline event.
 
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-data.ioos@noaa.gov.
+[INSERT CONTACT METHOD].
```
Comment (ocefpaf) [inline CODE_OF_CONDUCT.md]: copy-n-pasta

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+data.ioos@noaa.gov.
```
Comment (MathewBiddle) [inline CODE_OF_CONDUCT.md:63]: should be fixed here

**Comment by MathewBiddle** [comment]
@mwengren @ocefpaf How do you all feel about this? Please merge when ready.

**Review by ocefpaf — APPROVED** [review]
LGTM

**Comment by mwengren** [comment]
No issues here, merging now.

**Pull request merged**